ports/libxdf/vcpkg.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "libxdf",
+  "version-date": "2023-08-02",
```

It looks like the upstream version is 0.99.8. 
```suggestion
  "version": "0.99.8",
```

---

Hi! @JavierMatosD There was a bug in `0.99.8` and have been fixed by https://github.com/xdf-modules/libxdf/pull/38. That why I `upgrade` from `0.99.8` to latest commit. Without this fix, a 60~ LOC patch file is necessary.

---

If a stable release is preferred, then I can revert the last two commits of this PR.